Vadim_Shishlo/15.Technical_and_service_monitoring/monitoring/alertmanager.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-config
+  namespace: monitoring
+data:
+  config.yml: |-
+    global: {}
+    route:
+      receiver: 'slack_prometheus'
+      group_by: ['alertname', 'priority']
+      group_wait: 10s
+      repeat_interval: 20s
+      routes:
+        - receiver: slack_prometheus
+          match:
+            severity: slack
+          group_wait: 10s
+          repeat_interval: 20s
+    receivers:
+    - name: slack_prometheus
+      slack_configs:
+      - api_url: 'https://hooks.slack.com/services/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'
```

Comment:
Please remove, or cover this webhook url

Reply:
ok)